docs/competitors.md

```diff
@@ -1,33 +1,33 @@
 
 ## LGTM
 
-[LGTM](https://lgtm.co/docs/overview/) works in a very similar fashion as Zappr does. There are couple of things to note:
+[LGTM](https://lgtm.co/docs/overview/) works in a fashion very similar to Zappr. Some differences to note:
 
-* LGTM requires you to maintain a MAINTAINERS file to identify valid approvers. This is good practice anyways, but with Zappr you can reference entire Github organizations, collaborators of the project or single users.
-* LGTM gives you approvals for PRs. Zappr gives you that and also vetos, different approver groups, commit message checks, automatic branch creation, heuristics for specifications and more to come.
-* LGTM *"will require a [modest licensing fee for on-premise enterprise use](https://lgtm.co/docs/install/) in the near future"*. Zappr Enterprise you are allowed to just run without paying us.
-* LGTM has some [CLI tools](https://lgtm.co/docs/commands/). So [do](https://github.com/zalando/github-maintainer-cli) [we](https://github.com/zalando-incubator/oakkeeper).
+* LGTM requires you to keep a MAINTAINERS file to identify valid approvers. This is good practice anyways, but with Zappr you can reference entire GitHub organizations, project collaborators, or even single users.
+* LGTM gives you approvals for PRs. Zappr gives you that, as well as veto power for different approver groups, commit message checks, automatic branch creation, and heuristics for specifications, with more to come.
+* LGTM *"will require a [modest licensing fee for on-premise enterprise use](https://lgtm.co/docs/install/) in the near future"*. Zappr/Zappr Enterprise is completely free to use.
 * LGTM has a status page. [We don't](https://github.com/zalando/zappr/issues/357). 😕
+* LGTM uses [CLI tools](https://lgtm.co/docs/commands/) different from [do](https://github.com/zalando/github-maintainer-cli) [ours](https://github.com/zalando-incubator/oakkeeper).
 
 ## PullApprove
 
-[PullApprove](https://pullapprove.com) works also technically the same like Zappr. Noteworthy:
+[PullApprove](https://pullapprove.com) works, from a technical perspective, very much like Zappr does. Noteworthy differences:
 
-* PullApprove supports private repositores, currently we do not. We will support them in the future.
-* Some of PullApprove's features [cost money](https://pullapprove.com/pricing/), Zappr is 100 % free as in beer and freedom.
-* PullApprove has a custom web UI you can use to give approvals. Zappr strives to offer only the minimal necessary web UI as you should do your work on GitHub to avoid friction in your workflow.
-* As of now every customization available in PullApprove is also available in Zappr.
+* PullApprove supports private repositories, whereas currently we do not. We will support them in the future, though.
+* Some of PullApprove's features [cost money](https://pullapprove.com/pricing/). As stated above, Zappr is free.
```

Comment:
It's also a popular comparison when talking about OSS, to clarify whether you mean free as in "no money" or free as in "do what you want".